packages/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -143,10 +143,10 @@ export default class FocusZone extends React.Component<FocusZoneProps> implement
 
     if (!this._isInnerZone) {
       _outerZones.add(this)
-    }
 
-    if (this.windowElement && _outerZones.size === 1) {
-      this.windowElement.addEventListener('keydown', this._onKeyDownCapture, true)
+      if (this.windowElement && _outerZones.size === 1) {
```

Comment:
this looks great but i think possibly, you need to move the remove as well.

Reply:
done